services/app-api/forms/naaar.json

```diff
@@ -503,6 +506,494 @@
+                        "parentOptionId": "hdz3tEHdEBvRIIdgaovgEjaa"
+                      },
+                      "props": {
+                        "label": "Standard description",
+                        "hint": "Describe the standard (for example, 15 days)."
+                      }
+                    },
+                    {
+                      "id": "",
+                      "type": "checkbox",
+                      "validation": {
+                        "type": "checkbox",
+                        "nested": true,
+                        "parentFieldName": "standard_standardType",
+                        "parentOptionId": "hdz3tEHdEBvRIIdgaovgEjaa"
+                      },
+                      "props": {
+                        "label": "Analysis method(s) utilized to assess compliance for this standard for this program",
+                        "hint": "Select the method(s) utilized to assess compliance with the state's 42 C.F.R. § 438.68 and 42 C.F.R. § 438.206 standards. If a method is not listed here, add it in the “Analysis Methods” section.",
+                        "choices": []
+                      }
+                    }
+                  ]
+                },
+                {
+                  "id": "9eion8H8PNrCP0AeiikpsKbx",
+                  "label": "Service fulfillment",
+                  "children": [
+                    {
+                      "id": "QVdykzW594MqR5xWTrAF8elY",
+                      "type": "textarea",
+                      "validation": {
+                        "type": "text",
+                        "nested": true,
+                        "parentFieldName": "standard_standardType",
+                        "parentOptionId": "9eion8H8PNrCP0AeiikpsKbx"
+                      },
+                      "props": {
+                        "label": "Standard description",
+                        "hint": "Describe the standard (for example, 15 days)."
+                      }
+                    },
+                    {
+                      "id": "",
+                      "type": "checkbox",
+                      "validation": {
+                        "type": "checkbox",
+                        "nested": true,
+                        "parentFieldName": "standard_standardType",
+                        "parentOptionId": "9eion8H8PNrCP0AeiikpsKbx"
+                      },
+                      "props": {
+                        "label": "Analysis method(s) utilized to assess compliance for this standard for this program",
+                        "hint": "Select the method(s) utilized to assess compliance with the state's 42 C.F.R. § 438.68 and 42 C.F.R. § 438.206 standards. If a method is not listed here, add it in the “Analysis Methods” section.",
+                        "choices": []
+                      }
+                    }
+                  ]
+                },
+                {
+                  "id": "Uil8wXl6t3aBsI4W8pQZlO5P",
```

Comment:
Non-blocking: Are you generating this id as a UUID? Why not a more readable id?

Reply:
correct, we're generating these IDs as UUIDs because some of the nested choices will be have the exact same content under each option. ideally if a choice is pretty unique, we can give it a readable ID